src/common_main.c

```diff
@@ -67,21 +67,43 @@ static bool _setup_wally(void)
     return wally_set_operations(&_ops) == WALLY_OK;
 }
 
+// Go into bootloader if there was an error during startup, so a firmware update can be
+// applied. Otherwise, if there is an Abort() during startup, there would no way to reboot into the
+// bootloader and the device would be bricked.
+static void _bootloader_autoenter(void)
+{
+    auto_enter_t auto_enter = {
+        .value = sectrue_u8,
```

Comment:
Just a note: this is also set when in factorysetup. But I guess it's ok if it's set twice, although in factorysetup the upside_down is overwritten to `false`.

Reply:
Only on Abort() though. In that case the device is dead to be discarded, or it can be fixed, in which case it will run normally afterwards.